lhotse/cut/data.py

```diff
@@ -212,12 +212,17 @@ def move_to_memory(
 
             custom = {
                 # Case 1: Array
-                k: v.move_to_memory() if isinstance(v, Array)
-                # Case 2: TemporalArray
-                else v.move_to_memory(start=self.start, duration=self.duration)
-                if isinstance(v, TemporalArray)
-                # Case 3: anything else
-                else v
+                k: (
+                    v.move_to_memory()
+                    if isinstance(v, Array)
+                    # Case 2: TemporalArray
+                    else (
+                        v.move_to_memory(start=self.start, duration=self.duration)
+                        if isinstance(v, TemporalArray)
+                        # Case 3: anything else
+                        else v
+                    )
+                )
```

Comment:
This code shouldn't be reformatted, it's possible you're using a different version of black. Please revert all formatting changes.

Reply:
I'll continue to review once this is done (too many changes atm)

Reply:
Thanks @pzelasko , I have addressed your comments.